src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- Hi, Nice job so far. I would advise you not to disable the [linter](https://github.com/microverseinc/linters-config), because a linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Also, the project requirements prohibit disabling[ linter](https://github.com/microverseinc/linters-config). Kindly fix the linter manually and submit it again for re-review.